src/test/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/DataverehusRepositoryJdbcTest.java

```diff
@@ -40,6 +43,30 @@ public void tearDown() {
         cleanUpTestDb(namedParameterJdbcTemplate);
     }
 
+    @Test
+    public void hentSykefraværsstatistikkLand_lager_sum_og_returnerer_antall_tapte_og_mulige_dagsverk() {
+        insertSFStatLandInDvhTabell(namedParameterJdbcTemplate, 2018, 4, 5, 100);
```

Comment:
Sykefraværsstatistikk i stedet for SFStat? :P

Reply:
👍